model/src/main/java/hu/blackbelt/judo/meta/jsl/validation/JslDslValidator.xtend

```diff
@@ -1480,36 +1495,26 @@ class JslDslValidator extends AbstractJslDslValidator {
         }
     }
 
-    @Check
-    def checkForDuplicateNameForTransferMemberDeclaration(TransferMemberDeclaration member) {
-//    	if (!(member instanceof TransferFieldDeclaration || member instanceof TransferRelationDeclaration)) {
-//    		return
-//    	}
-    	
-        val TransferDeclaration transfer = member.parentContainer(TransferDeclaration)
-
-    	if (transfer === null) {
-    		return
-    	}
-
-        if (transfer.map !== null && member.name.toLowerCase.equals(transfer.map.name.toLowerCase)) {
-            error("Member declaration name conflicts with mapping field name: '" + member.name + "'.",
-                member.nameAttribute,
-                DUPLICATE_MEMBER_NAME,
-                member.name)
-        }
-		
-//		val members = transfer.eAllContents.filter[d | d instanceof TransferFieldDeclaration || d instanceof TransferRelationDeclaration].map[d | d as TransferMemberDeclaration].toList
-		val members = transfer.eAllContents.filter[d | d instanceof TransferMemberDeclaration].map[d | d as TransferMemberDeclaration].toList
-
-        if (members.filter[m | m.name.toLowerCase.equals(member.name.toLowerCase)].size > 1) {
-            error("Duplicate member declaration: '" + member.name + "'.",
-                member.nameAttribute,
-                DUPLICATE_MEMBER_NAME,
-                member.name)
-        }
-    }
-
+//    @Check
+//    def checkForDuplicateNameForTransferMemberDeclaration(TransferMemberDeclaration member) {
+//        val TransferDeclaration transfer = member.parentContainer(TransferDeclaration)
+//
+//        if (transfer.map !== null && member.name.toLowerCase.equals(transfer.map.name.toLowerCase)) {
+//            error("Member declaration name conflicts with mapping field name: '" + member.name + "'.",
+//                member.nameAttribute,
+//                DUPLICATE_MEMBER_NAME,
+//                member.name)
+//        }
+//		
+//		val members = transfer.eAllContents.filter[d | d instanceof TransferMemberDeclaration].map[d | d as TransferMemberDeclaration].toList
+//
+//        if (members.filter[m | m.name.toLowerCase.equals(member.name.toLowerCase)].size > 1) {
+//            error("Duplicate member declaration: '" + member.name + "'.",
+//                member.nameAttribute,
+//                DUPLICATE_MEMBER_NAME,
+//                member.name)
+//        }
+//    }
```

Comment:
What is the reason of commented out check? Where is the alternative?

Reply:
The "def checkForDuplicateName(Named named)" changed to check any Named declaration.
The commented out code is temporary and will be removed later.